src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/CrispImageSourceConverter.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Windows.Media;
+using Microsoft.VisualStudio.Imaging.Interop;
+using Microsoft.VisualStudio.PlatformUI;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.InheritanceMargin.MarginGlyph
+{
+    internal class CrispImageSourceConverter : MultiValueConverter<ImageMoniker, double, double, double, Color, ImageSource>
+    {
+        protected override ImageSource Convert(
+            ImageMoniker imageMoniker,
+            double height,
+            double width,
+            double dpi,
+            Color background,
+            object parameter,
+            CultureInfo culture)
+        {
+            var attributes = new ImageAttributes
```

Comment:
If we finally want to go this way, some further work needs to be done here to make sure zooming works. (Right now it doesn't zoom based on the zoom level)